frontend/src/utils/styles.js

```diff
@@ -26,21 +26,254 @@ const junctionStyle = makeStyles({
+    fs8: {
+        fontSize: theme.spacing(8),
+    },
+    fs8half: {
+        fontSize: theme.spacing(8.5),
+    },
+    fs9: {
+        fontSize: theme.spacing(9),
+    },
+    fs9half: {
+        fontSize: theme.spacing(9.5),
+    },
+    fs10: {
+        fontSize: theme.spacing(10),
+    },
     cardNote: {
         fontSize: '18px',
     },
+    m0: {
+        margin: theme.spacing(0),
+    },
+    mhalf: {
+        margin: theme.spacing(0.5),
+    },
+    m1: {
+        margin: theme.spacing(1),
+    },
+    m1half: {
+        margin: theme.spacing(1.5),
+    },
+    m2: {
+        margin: theme.spacing(2),
+    },
+    m2half: {
+        margin: theme.spacing(2.5),
+    },
+    m3: {
+        margin: theme.spacing(3),
+    },
+    m3half: {
+        margin: theme.spacing(3.5),
+    },
+    m4: {
+        margin: theme.spacing(4),
+    },
+    m4half: {
+        margin: theme.spacing(4.5),
+    },
+    m5: {
+        margin: theme.spacing(5),
+    },
+    m5half: {
+        margin: theme.spacing(5.5),
+    },
+    m6: {
+        margin: theme.spacing(6),
+    },
+    m6half: {
+        margin: theme.spacing(6.5),
+    },
+    m7: {
+        margin: theme.spacing(7),
+    },
+    m7half: {
+        margin: theme.spacing(7.5),
+    },
+    m8: {
+        margin: theme.spacing(8),
+    },
+    m8half: {
+        margin: theme.spacing(8.5),
+    },
+    m9: {
+        margin: theme.spacing(9),
+    },
+    m9half: {
+        margin: theme.spacing(9.5),
+    },
+    m10: {
+        margin: theme.spacing(10),
+    },
+    displayFlex: {
+        display: 'flex',
+    },
+    justifyContentStart: {
+        justifyContent: 'flex-start',
+    },
+    justifyContentEnd: {
+        justifyContent: 'flex-end',
+    },
+    justifyContentCenter: {
+        justifyContent: 'center',
+    },
+    justifyContentSpaceBetween: {
+        justifyContent: 'space-between',
+    },
+    justifyContentSpaceAround: {
+        justifyContent: 'space-around',
+    },
+    justifyContentSpaceEvenly: {
+        justifyContent: 'space-evenly',
+    },
+    alignItemsStart: {
+        alignItems: 'flex-start',
+    },
+    alignItemsEnd: {
+        alignItems: 'flex-end',
+    },
+    alignItemsCenter: {
+        alignItems: 'center',
+    },
+    alignItemsBaseline: {
+        alignItems: 'baseline',
+    },
+    alignItemsStretch: {
+        alignItems: 'stretch',
+    },
+    alignContentStart: {
+        alignContent: 'flex-start',
+    },
+    alignContentEnd: {
+        alignContent: 'flex-end',
+    },
+    alignContentCenter: {
+        alignContent: 'center',
+    },
+    alignContentSpaceBetween: {
+        alignContent: 'space-between',
+    },
+    alignContentSpaceAround: {
+        alignContent: 'space-around',
+    },
+    alignContentStretch: {
+        alignContent: 'stretch',
+    },
+    displayGrid: {
+        display: 'grid',
+    },
+    gridTemplateColumns: columnCount => ({
+        gridTemplateColumns: `repeat(${columnCount}, 1fr)`,
+    }),
```

Comment:
I tried using this classes but couldn't make it work without having to repeat the declarations of the junctionStyle() hook

I managed to make it work by changing it to

```
gridTemplateColumns: props => ({
        gridTemplateColumns: `repeat(${props.columnCount}, 1fr)`,
    }),
```

So then when I use junctionStyle inside a component, I can do a declaration like this:

```
const props = {
        columnCount: 3,
    }
    const classes = junctionStyle(props)
```

Which means that for properties that can be defined when declaring the property, like the columnCount or gap, you can just declare a `props` object with all those defined and then pass the `props` object as an argument to the `junctionStyle(props)`

Did you have a different way to define those arguments that maybe I missed? If not, I recommend you update the declarations that take arguments to the one I showed so we can avoid repeating the declarations of junctionStyle